Formula/torchvision.rb

```diff
@@ -75,7 +75,17 @@ def install
       'requires = ["setuptools", "torch", "wheel"]',
       'requires = ["setuptools", "wheel"]'
 
-    virtualenv_install_with_resources using: "python@3.11"
+    python3 = "python3.11"
+    venv = virtualenv_create(libexec, python3)
+    venv.pip_install resources
+
+    # We depend on pytorch, but that's a separate formula, so install a `.pth` file to link them.
+    # This needs to happen _before_ we try to install torchvision.
+    site_packages = Language::Python.site_packages(python3)
+    pytorch = Formula["pytorch"].opt_libexec
+    (libexec/site_packages/"homebrew-pytorch.pth").write pytorch/site_packages
```

Comment:
Try doing
```ruby
ENV.append_path "PYTHONPATH",pytorch/site_packages
```
before calling `virtual_env_install_with_resources`. _Then_ you can write the `.pth` file.

Reply:
Ugh, still not working. Any other suggestions? You're welcome to just push commits to this branch, as well.